### PR TITLE
Options footer

### PR DESCRIPTION
**Pull request by peppy**
image

### CODE REVIEW

File: osu.Game/Overlays/Options/OptionsFooter.cs [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+        private void load(OsuGameBase game, OsuColour colours)
+        {
+            RelativeSizeAxes = Axes.X;
+            AutoSizeAxes = Axes.Y;
+            Direction = FillDirection.Vertical;
+            Padding = new MarginPadding { Top = 20, Bottom = 30 };
+
+            var modes = new List<Drawable>();
+
+            foreach (PlayMode m in Enum.GetValues(typeof(PlayMode)))
```
Comment (smoogipooo) [inline osu.Game/Overlays/Options/OptionsFooter.cs]: Surround interior with { }

Comment (peppy) [inline osu.Game/Overlays/Options/OptionsFooter.cs]: nah. if we're going to do this here then it should be enforced solution-wide. not int he scope of this PR

File: osu.Game/OsuGameBase.cs [hunk 2]
```diff
@@ -37,6 +40,40 @@ public class OsuGameBase : Framework.Game, IOnlineComponent
...
+        protected AssemblyName AssemblyName => Assembly.GetEntryAssembly()?.GetName() ?? new AssemblyName() { Version = new Version() };
+
+        public bool IsDeployedBuild => AssemblyName.Version.Major > 0;
+
+        public bool IsDebug
+        {
+            get
+            {
+                bool isDebug = false;
+                Debug.Assert(isDebug = true);
```
Comment (Tom94) [inline osu.Game/OsuGameBase.cs]: I hope this is meant as a joke, because it did make me laugh. 😆

Comment (Tom94) [inline osu.Game/OsuGameBase.cs]: I guess we can keep it if it works. Hacky, but awesome.

File: osu.Game/OsuGameBase.cs [hunk 3]
```diff
@@ -37,6 +40,40 @@ public class OsuGameBase : Framework.Game, IOnlineComponent
...
+                return isDebug;
+            }
+        }
+
+        public string Version
+        {
+            get
+            {
+                bool isDebug = false;
+                Debug.Assert(isDebug = true);
```
Comment (Tom94) [inline osu.Game/OsuGameBase.cs]: Use `IsDebug` here.

**Pull request merged**